gold/testsuite/nanomips_fix_imm48.s

```diff
@@ -0,0 +1,43 @@
+	.linkrelax
+	.section	.text,"ax",@progbits
+	.align	4
+	.globl	__start
+	.ent	__start
+__start:
+	li $a0, foo
```

Comment:
Would be useful to have a comment for each case to describe what is being tested.